test/suites/basic.sh

```diff
@@ -369,7 +369,7 @@ test_basic_usage() {
   # make sure that privileged containers are not world-readable
   lxc profile create unconfined
   lxc profile set unconfined security.privileged true
-  lxc init testimage foo2 -p unconfined
+  lxc init testimage foo2 -p unconfined -s "lxdtest-$(basename "${LXD_DIR}")"
```

Comment:
The `unconfined` profile the container is attached to does not have a root device set.